plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/ITextRegionExtensions.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.formatting2.regionaccess;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.xtext.Action;
+import org.eclipse.xtext.ParserRule;
+import org.eclipse.xtext.RuleCall;
+
+/**
+ * @author Moritz Eysholdt - Initial contribution and API
+ */
+public interface ITextRegionExtensions {
```

Comment:
Discussed offline to keep it this way; Methods from the *Extensions interface are intended to be on Xtend's extension scope while most methods from *Access should be on the extensions scope and some not even on the implicit scope.